components/submit/blockstack-id-card.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Is this component going to be used? I don't see it anywhere yet - I'd assume maybe it'll end up in `MakerNav`

Reply:
used on `submit-your-app.tsx` page